### PR TITLE
The grand consolidation

### PR DESCRIPTION
**Pull request by kenotron**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-  "version": "2.1.0",
+  "name": "satcheljs-suite",
+  "version": "2.2.0",
   "description": "Store implementation for functional reactive flux.",
-  "main": "./dist/src/index.js",
   "scripts": {
-    "build": "node_modules/.bin/tsc",
-    "test": "npm run build && jasmine JASMINE_CONFIG_PATH=jasmine.json --verbose"
-  },
-  "author": "Scott Mikula <smikula@microsoft.com>",
```
Comment (MLoughry) [inline package.json]: Should "author" and "license" be deleted?

File: package.json [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
     "jasmine": "~2.4.1",
-    "typescript": "~1.8.9"
+    "mkdirp": "^0.5.1",
+    "rimraf": "^2.5.4",
+    "toposort": "~1.0.0",
+    "typescript": "~2.0.3"
   },
-  "typings": "./dist/src/index.d.ts",
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org/"
```
Comment (MLoughry) [inline package.json]: Did you mean to delete these config values as well?

File: package.json (hunk 1 above)
Comment (kenotron) [inline package.json]: license should be there still
but author can be deleted.

File: package.json (hunk 2 above)
Comment (kenotron) [inline package.json]: yup - we don't publish THIS repo, we use this repo to npm install globally for the entire set of pkgs

**Comment by kenotron** [comment]
Approved

**Pull request merged**